Building with JDK 8 or 9.

The method signature on sun.net.www.protocol.http.Handler now includes an IOException.

Which causes a build error.

```
[javac] /thebounty/work/redhat/java/faban/faban-git/driver/src/com/sun/faban/driver/transport/sunhttp/Handler.java:38: error: unreported exception IOException; must be caught or declared to be thrown
    [javac] 	    return new HttpURLConnection(u, p, this);
    [javac] 	           ^
    
```

--- a/driver/src/com/sun/faban/driver/transport/sunhttp/Handler.java
+++ b/driver/src/com/sun/faban/driver/transport/sunhttp/Handler.java
@@ -23,6 +23,7 @@
  */
 package com.sun.faban.driver.transport.sunhttp;
 
+import java.io.IOException;
 import java.net.Proxy;
 import java.net.URL;
 
@@ -34,7 +35,7 @@ import java.net.URL;
 public class Handler extends sun.net.www.protocol.http.Handler {
 
     @Override
-	protected java.net.URLConnection openConnection(URL u, Proxy p) {
+	protected java.net.URLConnection openConnection(URL u, Proxy p) throws IOException {
 	    return new HttpURLConnection(u, p, this);
     }
 }

